systems/tioga/compilers/rocm/00-rocm-551-compilers.yaml

```diff
@@ -20,8 +20,8 @@ compilers:
     modules: []
     environment:
       prepend_path:
-        LD_LIBRARY_PATH: /opt/cray/pe/cce/16.0.0/cce/x86_64/lib
-    extra_rpaths: [/opt/cray/pe/cce/16.0.0/cce/x86_64/lib/, /opt/cray/pe/gcc-libs/]
+        LD_LIBRARY_PATH: "/opt/cray/pe/cce/16.0.0/cce/x86_64/lib:/opt/rocm-5.5.1/lib"
```

Comment:
So this works on both the static and dynamic tioga?

Reply:
I will not work on static tioga. I'll need to add the `LD_LIBRARY_PATH`